internal/compiler/llr/lower_expression.rs

```diff
@@ -252,6 +252,10 @@ pub fn lower_expression(
             rhs: Box::new(lower_expression(rhs, ctx)),
         },
         tree_Expression::EmptyComponentFactory => llr_Expression::EmptyComponentFactory,
+        tree_Expression::DebugHook { expression, .. } => {
+            // Ignore debug-hoooks in LLR for now!
```

Comment:
```suggestion
```

No need to scream it ;-)  (the "for now!" is over the top and doesn't say much. Until when?)